rmf_robot_sim_gz_plugins/src/TeleportIngestor.cpp

```diff
@@ -248,7 +454,6 @@ void TeleportIngestorPlugin::init_non_static_models_poses(
 }
 
 TeleportIngestorPlugin::TeleportIngestorPlugin()
-: _ingestor_common(std::make_unique<TeleportIngestorCommon>())
 {
 }
```

Comment:
Now that the constructor does nothing we can safely remove it (or define it as `= default`).